pkg/storaged/fsys-tab.jsx

```diff
@@ -99,14 +96,24 @@ function nice_block_name(block) {
     return utils.block_name(client.blocks[block.CryptoBackingDevice] || block);
 }
 
-export function is_valid_mount_point(client, block, val) {
+export async function is_valid_mount_point(client, block, val, ignore_overmounting) {
     if (val === "")
         return _("Mount point cannot be empty");
 
     const other_blocks = find_blocks_for_mount_point(client, val, block);
     if (other_blocks.length > 0)
         return cockpit.format(_("Mount point is already used for $0"),
                               other_blocks.map(nice_block_name).join(", "));
+
+    if (!ignore_overmounting) {
+        const children = utils.find_children_for_mount_point(client, val, block);
+        if (Object.keys(children).length > 0)
+            return <>
+                {_("Filesystems are already mounted below this mountpoint.")}
+                {Object.keys(children).map(m => <div key={m}>{cockpit.format("• $0 on $1", nice_block_name(children[m]), m)}</div>)}
+                {_("Please unmount them first.")}
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19388-20230926-135906-360ec0bd-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)